docs/installing.md

```diff
@@ -1,73 +1,28 @@
+## Install on Ubuntu or Debian
```

Comment:
there is some content in the user guide doc before this. Please add that as well.

Reply:
fixed